install.html

```diff
@@ -7,9 +7,85 @@
 <script src=docs.js></script>
 
 <h1>Selenium Installation</h1>
+Selenium set up is quite different than other commercial tools. You don't install a tool using
+selenium binaries but set up your test automation project to use selenium binary in the langauge
+of your choice. The selenium binary you would need depends on browsers you want to run
+your tests upon.
```

Comment:
Well, then the word "installation" seems misleading to me. Those who come from commercial tools or manual testing background would find it surprising that installing selenium does not really install a GUI based tool.